Resources/Prototypes/Reagents/medicine.yml

```
@@ -18,6 +18,10 @@
   desc: A broad-spectrum anti-toxin, which treats toxin damage in the blood stream. Overdosing will cause vomiting, dizzyness and pain.
   physicalDesc: translucent
   color: "#3a1d8a"
+  Metabolism:
```

This needs to be lowercase `metabolism` or the serialization system wont read the property since it is case sensitive.

```suggestion
  metabolism:
```